cmd/krew/cmd/root.go

```diff
@@ -209,11 +210,21 @@ func cleanupStaleKrewInstallations() error {
 }
 
 func checkIndex(_ *cobra.Command, _ []string) error {
-	if ok, err := gitutil.IsGitCloned(paths.IndexPath(constants.DefaultIndexName)); err != nil {
-		return errors.Wrap(err, "failed to check local index git repository")
-	} else if !ok {
+	entries, err := ioutil.ReadDir(paths.IndexBase())
+	if err != nil {
+		return errors.Wrap(err, "failed to list directory")
+	}
+	if len(entries) == 0 {
 		return errors.New(`krew local plugin index is not initialized (run "kubectl krew update")`)
 	}
+	for _, e := range entries {
```

Comment:
I think we should add something like [this](https://github.com/kubernetes-sigs/krew/blob/master/internal/index/indexoperations/index.go#L46-L48) as a check here. We've seen before for mac users who get a `.DS_STORE` file in their krew directories (not sure how).